content/docs/command-reference/plots/index.md

```diff
@@ -96,7 +84,143 @@ names in the `train` array below:
 }
 ```
 
-## Plot templates (data series only)
+## Definining a plot
```

Comment:
"Configuring a plot"? WDYT?

It might help here to explain that you can specify a data file and configure the plot in the plots commands. Adding a plot to dvc.yaml saves you from having to repeat it and allows you to consistently view a specified set of plots. dvc.yaml can also persist your plots configuration and enables you to combine data from multiple files into one plot.

@jorgeorpinel Any thoughts/ideas on wording?

Reply:
> enables you to combine data from multiple files into one plot.

This is pretty much in the next paragraph... should I add it here too?

Reply:
Sorry @pared, my focus on this comment was on the benefits of putting plots configuration into `dvc.yaml`, regardless of whether they are standalone or stage plots.

On second thought, I think we should include `dvc.yaml` in the section title ("Configuring in `dvc.yaml`"?). Although plots may be configured from the CLI, neither the data sources nor the configuration get saved, and only a subset of plots configuration is actually supported in the current CLI (for example, plots with multiple data sources can't be configured in the CLl).

WDYT?

Reply:
> "Configuring a plot"? WDYT?

Sorry for the delay @dberenbaum (I didn't want to add yet another reviewer... but here we are).

Actually I think "defining" is better. But I'm about to make a suggestion on what to do with all these very long blocks of new content (may need another PR split TBH).